css/includes/components/_richtext.scss

```diff
@@ -54,9 +54,10 @@ body.mce-content-body {
     justify-content: flex-end;
 }
 
-.tox.tox-tinymce {
-    border: var(--tblr-border-width) solid var(--tblr-border-color);
+.tox.tox-tinymce:not(.content-editable-tinymce .tox.tox-tinymce) {
+    border: var(--tblr-border-width) solid var(--tblr-border-color) !important;
```

Comment:
Somewhat unrelated, I've noticed that some of tiynmce styles were not applied correctly (since I was trying to display a form field next to a tinymce fields, the differences were very noticeable).

Fixed it and made the selector more precise as we don't want to impact the special `.content-editable-tinymce` class.